lib/phoenix_live_view.ex

```diff
@@ -343,12 +344,19 @@ defmodule Phoenix.LiveView do
   defmacro __using__(opts) do
     # Expand layout if possible to avoid compile-time dependencies
     opts =
-      with true <- Keyword.keyword?(opts),
-           {layout, template} <- Keyword.get(opts, :layout) do
-        layout = Macro.expand(layout, %{__CALLER__ | function: {:__live__, 0}})
-        Keyword.replace!(opts, :layout, {layout, template})
+      if Macro.quoted_literal?(opts) do
```

Comment:
I * believe* this is compatible with the prior implementation for expanding the aliases.